src/pages/IPFSPin.vue

```diff
@@ -350,6 +371,70 @@ export default {
           path: this.cid_name
         })
       }
+    },
+    async putContent (message, content, inline_requested, api_server) {
+      let inline = inline_requested
+      if (inline) {
+        let serialized = JSON.stringify(content)
+        if (serialized.length > 150000) {
```

Comment:
The threshold is in bytes, not in character length. I think 150k characters (up to 600K bytes in the worst UTF-8 case) is a bit too high and the message will be rejected by the node. 
Did you test that your limit is actually working?

Reply:
I didn't write this code, I copied it directly from aleph-js (https://github.com/aleph-im/aleph-js/blob/359ecae06ac5580f07d5790a43ea3a119b607a3d/src/api/create.js#L14) and removed the storage part.
But I can decrease the max length